RELEASE-NOTES.md

```diff
@@ -0,0 +1,15 @@
+# v2.0.0
+
+## Backward incompatibility
+* In `snowpark function` command:
+  * Combined options `--function` and `--input-parameters` to `identifier` argument
+  * Changed name of option from `--return-type` to `returns`
+* In `snowpark procedure` command:
+  * Combined options `--procedure` and `--input-parameters` to `identifier` argument
+  * Changed name of option from `--return-type` to `returns`
+* In `snowpark procedure coverage` command:
+  * Combined options `--name` and `--input-parameters` to `identifier` argument
+
+## New additions
```

Comment:
Can we create a task to add entries for tasks which we have already completed after the last release?

It would simplify the next release.

Reply:
Added